packages/temen/src/throttle/index.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * throttle 입니다.
+ * @param {Function} callback - delay(지연) 이후에 실행되는 함수입니다.
+ * @param {number} delay - delay(지연)시킬 시간 millisecond 기준.
+ * @example
+ * ```ts
+ *  const callbackFunc = () => {
+ *    return 'callback'
+ *  }
+ *
+ *  const throttle(callback, 300);
+ * ```
+ */
+
+function throttle<T extends unknown[]>(callback: (...args: T) => void, delay: number) {
+  let isThrottle: boolean;
```

Comment:
```suggestion
  let isThrottle = false;
```
혹시 여기 이렇게 초기화해주면 안되는 건가유?? 👀

Reply:
오! 아니요 당연히 됩니다 false로 해주는게 훨씬 명확하겠네요 요상한 습관이 었읏십니다 요렇게 수정할게요! 피드백 감사합니다 🙏

Reply:
말씀해주신 부분 아래와 함께 35e7829a079d6c4e587c7e6a9090622f61db89ef 반영했습니다! 🙏